state.js

```diff
@@ -2,25 +2,99 @@
 const state = {
   isWorkInterval: true,
   timerIntervalId: 0,
-  todoList: [],
-  maxId: 0,
+  todoList: [
+    {
+      todoText: "Test",
+      priority: "High",
+      id: 1,
+      endDay: new Date("2024-05-01").toLocaleDateString(),
+    },
+    {
+      todoText: "Test 1",
+      priority: "Low",
+      id: 2,
+      endDay: new Date("2024-05-02").toLocaleDateString(),
+    },
+  ],
+  maxId: 3,
   currentId: 0,
   currentTodo: 0,
   currentTodoColumn: 0,
-  currentTomatoesTime: 0,
   timer: {
     /** when we pause the timer, store current time for resuming */
     remainingTimeOnResume: undefined,
+    currentTomato: undefined,
+    today: new Date(),
+    currentMonth: new Date().getMonth(),
+    currentYear: new Date().getFullYear(),
+    currentDay: new Date().getDate(),
+    // stored tomatoes month - index.
+    tomatoes: [
```

Comment:
```suggestion
    tomatoes: INITIAL_TOMATOES
```

same as above, maybe cleaner